runtime/src/main/java/io/quarkiverse/openapi/generator/CodegenConfig.java

```diff
@@ -0,0 +1,43 @@
+package io.quarkiverse.openapi.generator;
+
+import java.util.Map;
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.common.utils.StringUtil;
+
+/**
+ * This class represents the runtime configurations for the openapi-generator extension.
+ */
+@ConfigRoot(name = CodegenConfig.RUNTIME_TIME_CONFIG_PREFIX, phase = ConfigPhase.RUN_TIME)
+public class CodegenConfig {
```

Comment:
This class must be renamed (maybe `OpenApiGeneratorConfig`). There's no codegen config in runtime.